client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -198,6 +211,28 @@ export class BpmnEditor extends CachedComponent {
     }
   }
 
+  async loadTemplates() {
+    const { getConfig } = this.props;
+
+    const modeler = this.getModeler();
+
+    const templatesLoader = modeler.get('elementTemplatesLoader');
+
+    let templates = await getConfig('bpmn.elementTemplates');
+
+    templatesLoader.setTemplates(getCloudTemplates(templates));
+
+    const propertiesPanel = modeler.get('propertiesPanel', false);
```

Comment:
When do we have no properties panel?

Reply:
This probably never happens, I simply copy-pasted it from the other bpmn editor.

😄

Reply:
In that case, I'd suggest to remove it. By the way, I'd expect the properties panel to know that it should update when templates are loaded, which [it should already do](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/b5a29d41d5493d3a35be9b19a961257d46def4af/src/render/BpmnPropertiesPanel.js#L143).

Reply:
I will remove this. Furthermore, the properties panel doesn't even have an `update` API, that is legacy code that we can safely remove.

Reply:
Let's merge the PR as soon as you remove this.